### PR TITLE
Improvements

### PR DESCRIPTION
**Pull request by Arpit-SerpentCS**

### CODE REVIEW

File: base_synchro/README.rst [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+Bug Tracker
+===========
+
+Credits
+=======
+
+Contributors
+------------
+
+* Jay Vora <jay.vora@serpentcs.com>
```
Comment (JayVora-SerpentCS) [inline base_synchro/README.rst]: Serpent Consulting Services Pvt. Ltd.

File: base_synchro/README.rst [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+   :alt: Odoo Community Association
+   :target: http://odoo-community.org
+
+This module is maintained by the OCA.
+
+OCA, or the Odoo Community Association, is a nonprofit organization whose
+mission is to support the collaborative development of Odoo features and
+promote its widespread use.
+
+To contribute to this module, please visit http://odoo-community.org.
```
Comment (JayVora-SerpentCS) [inline base_synchro/README.rst]: OCA is not the maintainer here.

File: base_synchro/models/base_synchro_obj.py [hunk 3]
```diff
@@ -45,15 +44,16 @@ class BaseSynchroObj(models.Model):
...
+#    def get_ids(self, obj, dt, domain=None, action=None):
+#        """Method to get ids."""
+#        if action is None:
+#            action = {}
+#        return self._get_ids(obj, dt, domain, action=action)
 
+#    def _get_ids(self, obj, dt, domain=None, action=None):
     @api.model
-    def _get_ids(self, obj, dt, domain=None, action=None):
+    def get_ids(self, obj, dt, domain=None, action=None):
```
Comment (JayVora-SerpentCS) [inline base_synchro/models/base_synchro_obj.py]: why removing? We hav both the methods being used. isnt it?

Comment (Arpit-SerpentCS) [reply base_synchro/models/base_synchro_obj.py]: _get_ids method is called from  synchronize method. And get_ids is called from _get_ids. So I have removed _get_ids method and directly called get_ids method.

**Comment by JayVora-SerpentCS** [comment]
Fix travis

**Pull request merged**